File: src/main/java/nl/tudelft/ewi/se/in4355/server/jobs/Count.java
package nl.tudelft.ewi.se.in4355.server.jobs;

class Count{
	public final String word;
	public final int count;
	
	public Count() {
		this("", -1);
	}
	
	public Count(String word, int count) {
		this.word = word;
		this.count = count;
	}
}

File: src/main/java/nl/tudelft/ewi/se/in4355/server/Count.scala
package nl.tudelft.ewi.se.in4355.server

import org.codehaus.jackson.annotate.JsonCreator
import org.codehaus.jackson.annotate.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class Count(var word: String, var count: Integer) {
    @JsonCreator
    def this() {
    	this(word = "", count = 0);
    }
};